base/shared/src/main/scala/scalaz/data/kleisli.scala

```diff
@@ -18,8 +18,12 @@ sealed trait KleisliModule {
 
   def second[F[_], A, B, C](k: Kleisli[F, A, B])(implicit F: Functor[F]): Kleisli[F, (C, A), (C, B)]
 
+  def dimap[F[_], A, B, C, D](k: Kleisli[F, A, B])(ca: C => A)(bd: B => D)(implicit F: Functor[F]): Kleisli[F, C, D]
+
   def compose[F[_], A, B, C](j: Kleisli[F, B, C], k: Kleisli[F, A, B])(implicit B: Bind[F]): Kleisli[F, A, C]
 
+  def lift[F[_], A, B](fb: F[B]): Kleisli[F, A, B]
```

Comment:
Not sure `lift` is a good name. What about `const`?

Reply:
```scala
Function.const[B, A]   :   B  => (A =>   B )
Kleisli.const[F, A, B] : F[B] => (A => F[B])
```

Reply:
`lift` here is by analogy to every other monad transformer.

Reply:
@TomasMikula feel free to smack me with a newspaper when i'm being dumb 😄 

That said... _eehhhh_ `lift` still sounds a little better

Reply:
Yeah, I think I'm with @TomasMikula here; `lift` seems more apt for lifting a function rather than a wrapped value.

Reply:
That's fair, didn't think of separating ReaderT and Kleisli.